internal/pkg/deploy/cloudformation/stack/worker_svc.go

```diff
@@ -135,6 +135,7 @@ func (s *WorkerService) Template() (string, error) {
 		CredentialsParameter:     aws.StringValue(s.manifest.ImageConfig.Credentials),
 		ServiceDiscoveryEndpoint: s.rc.ServiceDiscoveryEndpoint,
 		Subscribe:                subscribe,
+		Platform:                 convertPlatform(s.manifest.Platform),
```

Comment:
This field won't be surfaced for App Runner services, but if users do throw this field into their manifest, we should handle it, right?

Reply:
Mmm good question! 

💭  For app runner services, maybe one option is to handle that in `UnmarshalYAML`? so that if they try to set anything that's not `linux/amd64` or `linux/x86_64` we throw an error out. Right now it will use whatever value in `platform` to strictly build the container image `docker build --platform <val>`but won't pass it down to the template.

like this:
```go
type appRunnerPlatform struct {
   *PlatformArgsOrString // embeded
}

type ecsPlatform  struct {
   *PlatformArgsOrString // embeded
}
```

Reply:
I'll add this to my next PR.